build/basics.tcl

```diff
@@ -99,7 +99,7 @@ expect ":KILL"
 respond "*" ":job peek\r"
 respond "*" ":load sysbin;peek bin\r"
 respond "*" "purify\033g"
-respond "Idle" "q"
+respond {$G} "q"
```

Comment:
What's "{$G}"?

Reply:
`$G` is the output from DDT.  `{` and `}` is TCL syntax for a string that doesn't do $ variable expansion (but keeps track of balanced braces).

Reply:
Ah, thanks.  Didn't know that.